crates/subspace-farmer/src/plotting.rs

```diff
@@ -116,7 +160,7 @@ pub(crate) async fn background_plotting<P: AsRef<[u8]>>(
                             if let Err(error) =
                                 subspace_solving.encode(piece_index_offset + position as u64, piece)
                             {
-                                error!("Failed to encode a piece: error: {}", error);
+                                error!("Failed to encode a piece: error: {:?}", error);
```

Comment:
Why?

Reply:
Intellij Idea was complaining it has not implemented display. This prevented red squiqlies. Nothing important, can be reverted.

Reply:
IDE might have false-positives, some of them will lead you to dead ends, no need to pretty print it here